GliaWidgets/View/Chat/Message/ChoiceCardStyle.swift

```diff
@@ -1,9 +1,21 @@
 import UIKit
 
+/// Style of the choice card sent to the visitor by the AI engine.
```

Comment:
@gersonnoboa - can you confirm that "choice cards" here are actually the same thing we call "response cards" elsewhere? (Here: https://docs.glia.com/glia-dev/docs/android-single-choice-messages#sending-the-visitors-response)
If so, we should change that in all the code comments.

Reply:
@gersonnoboa ping :)

Reply:
Yes, these are response cards. Sorry there are so many emails about this that I just didn't notice.